javascript/signin.js

```diff
@@ -0,0 +1,82 @@
+import { 
+  loginId,
+  emailNone,
+  emailFail,
+  emailCheck,
+  emailResult,
+  /* errorMessageEmailReset, */
+} from "./email.js"
+
+import { 
+  loginPwd,
+  passwordNone,
+  passwordFail,
+  passwordResult,
+  /* errorMessagePasswordReset, */
+} from "./password.js"
+
+const loginBtn = document.querySelector('.cta');
+const eyeBtn = document.querySelector('.password-button')
+const eyeOn = document.querySelector('.eye-on')
+const eyeOff = document.querySelector('.eye-off')
+
+loginId.addEventListener('focusout', emailResult);
+loginPwd.addEventListener('focusout', passwordResult);
+
+loginId.addEventListener('keyup', function(e) {
+  if (e.keyCode === 13) {
+    if (loginId.value === "test@codeit.com" && loginPwd.value === 'codeit101') {
+      location.href = "./folder.html";
+    } else {
+      emailNone.classList.add("hide");
+      emailCheck.classList.add("hide");
+      emailFail.classList.remove("hide");
+      loginId.classList.add("input-err");
+      passwordNone.classList.add("hide");
+      passwordFail.classList.remove("hide");
+      loginPwd.classList.add("input-err");
+    }
+  }
+}); /* 합칠 수 있나? */
+
+loginPwd.addEventListener('keyup', function(e) {
+  if (e.keyCode === 13) {
+    if (loginId.value === "test@codeit.com" && loginPwd.value === 'codeit101') {
```

Comment:
`test@codeit.com`, `codeit101` 와 같이 항상 값이 고정되어있는 것을 '상수' 라고 합니다
상수가 코드 안에 섞이게 되면 이후 상수를 변경해야 할 때 찾기 힘들기도 하고 관리가 어려워서 
보통 상수는 별도로 분리합니다. 
상수는 대문자 + snake_case 로 작성합니다 -> `SNAKE_CASE` <- 이렇게

예를 들어, 
```
const VALID_EMAIL = 'test@codeit.com';
const VALID_PASSWORD = 'codeit101';
```
와 같이 파일 상단 또는 별도 파일에 작성해두고 여기서는 
```suggestion
    loginId.value === VALID_EMAIL &&
    loginPwd.value === VALID_PASSWORD
```

로 작성할 수 있습니다!

(참고로 여기서 valid 라는 단어는 '유효한', '통과 가능한' 의 의미로 사용했습니다)